### PR TITLE
Get for ConfigMap

### LINKED ISSUE #15: Support getting a single configmap
#2 introduced configmap creation and enumeration.  Add an API which allows a single namespace to be retrieved by name (and namespace, presumably?  dig into that a little bit more).

### PR DESCRIPTION
**Pull request by exarkun**
Fixes #15

### CODE REVIEW

File: src/txkube/_memory.py [hunk 1]
```diff
@@ -132,7 +137,8 @@ def _delete(self, request, collection, collection_name, name):
     app = Klein()
     @app.handle_errors(NotFound)
     def not_found(self, request, name):
-        request.responseHeaders.setRawHeader(u"content-type", [u"application/json"])
+        # XXX Untested - https://github.com/LeastAuthority/txkube/issues/42
+        request.responseHeaders.setRawHeaders(u"content-type", [u"application/json"])
```
Comment (rodrigc) [inline src/txkube/_memory.py:143]: Adding a test here as specified in https://github.com/LeastAuthority/txkube/issues/42 would be nice

File: src/txkube/_memory.py [hunk 2]
```diff
@@ -100,18 +100,23 @@ class _Kubernetes(object):
...
+
     def _list(self, request, namespace, collection):
         if namespace is not None:
-            # Unfortunately pset does not support transform. :( Use this more
-            # verbose .set() operation.
-            collection = collection.set(
-                u"items",
-                pset(obj for obj in collection.items if obj.metadata.namespace == namespace),
-            )
+            collection = self._reduce_to_namespace(collection, namespace)
```
Comment (rodrigc) [inline src/txkube/_memory.py:108]: Might be nice to add a test for this.

**Review by rodrigc — APPROVED** [review]
Looks OK, more tests can be added later.

File: src/txkube/_memory.py (hunk 1 above)
Comment (exarkun) [reply src/txkube/_memory.py:143]: For sure, eventually.  Filed #42 so I could skip it for now. :)

**Pull request merged**